Data/Sys/GameSettings/GP7E01.ini

```
@@ -157,280 +838,241 @@ c24e6288 00000006
+$Mechanics - Flower Orb Gives 1 Coin Per Space [Airsola]
+041c4f24 3b400001
 
-$Game Mechanics - Microphone Always Off
-04098fc4 3800002c
-04098fc8 980317e6
-04098fcc 38600000
-*This cheat makes the microphone game setting stay off while playing Mario Party 7.
+$Mechanics - Fireball Orb Steals 20 Coins [Airsola]
+041c148c 38a0ffec
+041c1464 3b800014
 
-$Game Mechanics - Orbs Can Only Be Placed On Your Current Space
-041E7128 60000000
-*This cheat makes it so Orbs are most balanced. To be fair a human cannot throw that far.
+$Mechanics - Free Stars [gamemasterplc]
+04188774 3b800000
+0418876c 3b800000
+204ed598 40800030
+044ed598 48000030
+044ed754 38800000
+e2000001 80008000
+204f62c0 38030001
+044f62c4 38000000
+044f690c 38600000
+e2000001 80008000
+*Stars cost nothing instead of the usual 20 coins.
 
-$Game Mechanics - Replace "Nothing" Duel Result With "Half Coins"
-c21d8ed0 00000003
+$Mechanics - Hammer Bro Orb Steals 20 Coins [Ralf]
+041A902C 3AC00014
+041A9A28 38A0FFEC
+
+$Mechanics - Happening Star Becomes Star Star [Airsola]
+204e4ae0 48000210
+044e4a38 48000014
+044e4a6c 38000000
+e2000001 80008000
+204e16ec 8903001f
+044e16ec 8903003b
+e2000001 80008000
+204e1a0c 8903001f
+044e1a0c 8903003b
+e2000001 80008000
+*The Happening Star will become a bonus star awarded to the player who held the highest amount of stars at any point of the game. Recommended only for Windmillville, or Pyramid Park.
+*This Cheat is incompatible with Fixed Bonus Star because it implements it.
+
+$Mechanics - Improved Duel Results [Airsola]
+c21d8ed0 00000007
+2c030001 4082000c
+38600000 48000020
+2c030005 4082000c
+38600004 48000010
 2c030003 40820008
-38600001 7c711b78
+38600000 7c711b78
 60000000 00000000
-*This cheat makes it so duels don't end in a dud.
+041d8f68 3aa00028
+*(X) -> 40 Coins, 10 Coins -> 40 Coins, Half Coins -> 40 Coins, 2 Stars -> 1 Star
+
+$Mechanics - Last 5 Turns Event Is Always x3 Coins on Spaces
+042311A8 38000000
+
+$Mechanics - Last 5 Turns Event Is Always 10 More Koopa Kid Spaces
+042311A8 38000001
+
+$Mechanics - Last 5 Turns Event Is Always 40 Coin Bonus
+042311A8 38000002
+
+$Mechanics - Last 5 Turns Event Is Always Stars are 10 Coins
+042311A8 38000003
+
+$Mechanics - Last 5 Turns Event Is Always Red Spaces are Bowser Spaces
+042311A8 38000004
+
+$Mechanics - Last 5 Turns Event Is Always Chain Chomp Ride for 5 Coins (Pyramid Park)
+042311A8 38000005
+*Only use on Pyramid Park
+
+$Mechanics - Mic Minigames Have A 10% Chance Of Appearing Instead Of 20% [gamemasterplc]
+042f7a04 41200000
+*Less Mic Minigames and see more standard minigames.
 
-$Game Mechanics - Slow 'Shroom Orb Rolls 1-5
+$Mechanics - Microphone Always Off [gamemasterplc]
+04098fc4 3800002c
+04098fc8 980317e6
+04098fcc 38600000
+*Microphone game setting stay off while playing Mario Party 7.
+
+$Mechanics - Orbs Can Only Be Placed On Your Current Space [gamemasterplc]
+041e7128 60000000
+*Orb is severly balanced.
```

```suggestion
*Orb is more aggressively balanced.
```
I think this is what you mean here.